frontend/src/metabase/ui/components/inputs/MultiAutocomplete/MultiAutocomplete.module.css

```diff
@@ -1,33 +1,21 @@
 .pill {
-  --multiselect-pill-font-size: 0.875rem;
-}
-
-.pillList {
-  padding-right: var(--input-right-section-width);
-}
+  cursor: default;
 
-.input {
-  --input-fz: 0.875rem;
-
-  input::placeholder {
-    color: var(--mb-color-text-tertiary);
+  &:hover {
+    background-color: var(--mb-color-brand-alpha-88);
   }
 }
 
-.empty {
-  font-size: 0.875rem;
-}
-
-.optionList {
-  padding: 0.75rem;
-}
+@supports (field-sizing: content) {
```

Comment:
Can we solve this without `field-sizing`, or at least provide a better fallback? The current support is only [72%](https://caniuse.com/?search=field-sizing) and it looks weird without it

<img width="421" alt="Screenshot 2025-04-08 at 16 34 53" src="https://github.com/user-attachments/assets/ad8c688e-a1b6-4574-b0c0-08b57b636722" />

I feel like this should be solvable with flex, but if it's too hard, ignore this comment.

Reply:
This is hard to implement, flex won't work. We need to make the text input absolutely positioned and add a hidden span to take the space.

Reply:
really ideally I'd like to wait for the problem to "fix itself" if possible.